.git-blame-ignore-revs

```diff
@@ -65,3 +65,8 @@ d800b5fb74c077522ef7a0752e49968df46e745e
 edd9d0e0ef6898a0528ac6389e93e9fca6132ff6
 ee61bd6f2e4db8df8e92be6f92c02b808aa92234
 ef85b9af45922c6ca364930d40f8aacf1a6dfe64
+
+# Coding Style Conformance
+f07e8fed1c3caa78206f656292c270c74de984b8
+91c50e731e1307b52dfbba58bc1c376d54f8f723
+f6d3f9b771b947c14291467deb7dc46692e73d19
```

Comment:
These two commits should not be here. 

.git-blame-ignore-revs is intended for hiding mass-reformatting changes, and the latter two are well under the threshold to not be considered "mass".

This file is a tool of last-resort; It should not be used to needlessly hide history.